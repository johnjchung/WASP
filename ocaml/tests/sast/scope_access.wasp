Func int access() {
    int a = 5;
    if (true) {
        int i = 0;
        a = i;
    }
    return a;
}

Endpoint temp(GET:) {
    int a = access();
}
