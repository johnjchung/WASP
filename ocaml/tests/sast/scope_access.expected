Endpoint temp( GET:) {int a = [int] access();}Func int access( ) {
	int a = [int] 5;
if ([bool] true)
 {	int i = [int] 0;
	a = [int] i;
}

	Return [int] a;

}
